### ISSUE TITLE
A trailing slash is being added to the paths in additionalPaths even though config is set to `false`

### ISSUE THREAD

**Issue by dominicarrojado** [iamvishnusankar/next-sitemap#311]
**Describe the bug**
A trailing slash is being added to the paths in additionalPaths even though config is set to `false`. Seems like `next.config.js` is overriding it instead of the other way around. I [previously created bug reports](https://github.com/iamvishnusankar/next-sitemap/issues/258 & https://github.com/iamvishnusankar/next-sitemap/issues/223) but it was closed, assuming it was fixed, I tried with the latest version but still experience the same behaviour so recreating the issue again.

**To Reproduce**

1. Update `next.config.js`:
```
module.exports = {
  reactStrictMode: true,
  trailingSlash: true
};
```
2. Update `next-sitemap.js`:
```
module.exports = {
  siteUrl: 'https://example.com',
  generateRobotsTxt: true,
  trailingSlash: false,
  additionalPaths: async (config) => [
    await config.transform({...config, trailingSlash: false}, '/additional-page.html'),
  ],
};

```
3. Build and export next app
4. Result:
```
<url><loc>https://example.com/additional-page.html/</loc><changefreq>daily</changefreq><priority>0.7</priority><lastmod>2021-10-15T07:05:21.440Z</lastmod></url>
```

Code Sandbox: https://codesandbox.io/s/interesting-tu-toztvx?file=/public/sitemap-0.xml

**Expected behavior**
It shouldn't include trailing slash if not needed - `<loc>https://example.com/additional-page.html</loc>`. When crawlers access the page with trailing slash (e.g. `https://example.com/additional-page.html/`), the page will break because of the assets it requires. It should be accessed by crawlers without the trailing slash.

**Additional context**
I'm using `next-sitemap@2.5.1`.

**Comment by iamvishnusankar** [comment]
@dominicarrojado Fixed: #314

**Comment by dominicarrojado** [comment]
Thank you @iamvishnusankar! Works perfectly now
